.env.example

```diff
@@ -4,4 +4,6 @@ POSTGRESDB_DATABASE=
 POSTGRESDB_LOCAL_PORT=
 POSTGRESDB_DOCKER_PORT=
 SPRING_LOCAL_PORT=
-SPRING_DOCKER_PORT=
\ No newline at end of file
+SPRING_DOCKER_PORT=
+JWT_SECRET_KEY=
+JWT_EXPIRATION_TIME=
```

Comment:
You can send us example keys that your using so that we know te format

Reply:
No, security issues - time in milisec remember that name